### PR TITLE
Flask web

### PR DESCRIPTION
**Pull request by rpdavila**
Added bootstrap to img-upload.html giving the flash notification a nice UI feel
added flash notification with categories to views.py

### CODE REVIEW

File: web/ascii_art_web/templates/img-upload.html [hunk 1]
```diff
@@ -5,10 +5,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>image to ascii converter</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```
Comment (ndeoligence) [inline web/ascii_art_web/templates/img-upload.html:8]: Unclear what this is for?

File: web/ascii_art_web/templates/img-upload.html [hunk 2]
```diff
@@ -29,6 +45,8 @@ <h1>ASCII ART GENERATOR</h1>
             </div>
         </div>
     </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```
Comment (ndeoligence) [inline web/ascii_art_web/templates/img-upload.html:48]: These too?

File: web/ascii_art_web/templates/img-upload.html [hunk 3]
```diff
@@ -5,10 +5,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>image to ascii converter</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
 </head>
 
 <body>
+    <main role="main" class="container">
```
Comment (ndeoligence) [inline web/ascii_art_web/templates/img-upload.html:13]: The main tag messes up with existing page's styling. Not sure if it's just a me-thing?
How does it appear on your side?

**Review by ndeoligence — CHANGES_REQUESTED** [review]
Hi @rpdavila , please check out the added comments

File: web/ascii_art_web/templates/img-upload.html (hunk 1 above)
Comment (rpdavila) [reply web/ascii_art_web/templates/img-upload.html:8]: that's the bootstrap css linking to the cdn

File: web/ascii_art_web/templates/img-upload.html (hunk 2 above)
Comment (rpdavila) [reply web/ascii_art_web/templates/img-upload.html:48]: same the bootstrap javascript linking to the CDN

File: web/ascii_art_web/templates/img-upload.html (hunk 3 above)
Comment (rpdavila) [reply web/ascii_art_web/templates/img-upload.html:13]: you should be able to override the bootstrap css with the static style sheet that's why I put the bootstrap above the custom style sheet you have

File: web/ascii_art_web/templates/img-upload.html (hunk 1 above)
Comment (ndeoligence) [reply web/ascii_art_web/templates/img-upload.html:8]: I understand what the links do. I was asking why you include Bootstrap at all? It's quite a big file, and it didn't seem like you were using it.

But after another look at the html, I see you reference its classes. Sorry.

**Comment by rpdavila** [comment]
@ndeoligence I made a whole bunch of changes. Please take a look. Also fixed the trailing comma issue.

**Pull request merged**